tests/cli.js

```diff
@@ -488,3 +488,9 @@ test('run `cli --quiet` on 1 non-json file', macro.testCLI, {
   args: ['*/package.json', '--quiet'],
   message: 'Should output error message',
 })
+
+test('run `cli --stdin` with input from stdin', macro.testCLI, {
+  args: ['--stdin'],
+  message: 'Should output sorted json',
+  stdin: `{\n  "description": "Description",\n  "name": "Name"\n}\n`,
```

Comment:
Let's use `\r\n' to test. The line feed should be CRLF in output.

Reply:
Check [keithamus/sort-package-json@`605a73b` (#296)](https://github.com/keithamus/sort-package-json/pull/296/commits/605a73bc5e8b3554baed0bed2e4eaea1e50f8d21)

Reply:
Seems we can't print CRLF?

Reply:
Working with suggested changes https://github.com/keithamus/sort-package-json/pull/296#discussion_r1170896656

see [`4baabc7` (#296)](https://github.com/keithamus/sort-package-json/pull/296/commits/4baabc737874c14a828873532250cc53f637390b)